streams/resources/test-utils.js

```diff
@@ -1,50 +1,14 @@
 'use strict';
 
-self.getterRejects = (t, obj, getterName, target) => {
```

Comment:
Drive-by fix: these helpers are no longer used, since we now have the IDL harness tests. So I got rid of them. 😁